

@testset "Error" begin

    # default behavior: throw EnduserError
    @test begin
        try
            erroruser("foo")            
        catch e
            isa(e, EnduserError)
        end
    end

end

